armi/materials/thoriumOxide.py

```diff
@@ -16,9 +16,6 @@
 Thorium Oxide solid ceramic.
 
 Data is from [IAEA-TECDOCT-1450]_.
-
-.. [IAEA-TECDOCT-1450] Thorium fuel cycle -- Potential benefits and challenges, IAEA-TECDOC-1450 (2005).
-    https://www-pub.iaea.org/mtcd/publications/pdf/te_1450_web.pdf
```

Comment:
I don't understand this change.

This PDF link is still valid. And it is referenced as a data source in the line above.

We can't just delete this and lose the reference.